src/Products/Annotation/Annotator/DistanceAnnotator.cs

```diff
@@ -1,81 +1,95 @@
 {
-    public class DistanceAnnotator : AbstractBoxAnnotator
+    public class DistanceAnnotator : BaseAnnotator
     {
+        private DistanceAnnotation distanceAnnotation;
 
         public DistanceAnnotator(AnnotationDataEntity annotationData, PageData pageData)
             : base(annotationData, pageData)
         {
+            distanceAnnotation = new DistanceAnnotation()
+            {
+                Box = GetBox()
+            };
         }
 
-        public override AnnotationInfo AnnotateWord()
+        public override AnnotationBase AnnotateWord()
         {
-            AnnotationInfo distanceAnnotation = InitAnnotationInfo();
+            distanceAnnotation = InitAnnotationBase(distanceAnnotation) as DistanceAnnotation;
             return distanceAnnotation;
         }
 
-        public override AnnotationInfo AnnotatePdf()
+        public override AnnotationBase AnnotatePdf()
         {
-            AnnotationInfo distanceAnnotation = InitAnnotationInfo();
+            distanceAnnotation = InitAnnotationBase(distanceAnnotation) as DistanceAnnotation;
             return distanceAnnotation;
         }
 
-        public override AnnotationInfo AnnotateCells()
+        public override AnnotationBase AnnotateCells()
         {
-            throw new NotSupportedException(String.Format(Message, annotationData.type));
+            throw new NotSupportedException(string.Format(Message, annotationData.type));
         }
 
-        public override AnnotationInfo AnnotateSlides()
+        public override AnnotationBase AnnotateSlides()
         {
-            throw new NotSupportedException(String.Format(Message, annotationData.type));
+            throw new NotSupportedException(string.Format(Message, annotationData.type));
         }
 
-        public override AnnotationInfo AnnotateImage()
+        public override AnnotationBase AnnotateImage()
         {
-            // init annotation object
-            AnnotationInfo distanceAnnotation = InitAnnotationInfo();
-            distanceAnnotation.BackgroundColor = 15988609;
+            distanceAnnotation = InitAnnotationBase(distanceAnnotation) as DistanceAnnotation;
             return distanceAnnotation;
         }
 
-        public override AnnotationInfo AnnotateDiagram()
+        public override AnnotationBase AnnotateDiagram()
         {
-            // init annotation object
-            AnnotationInfo distanceAnnotation = InitAnnotationInfo();
-            distanceAnnotation.BackgroundColor = 15988609;
+            distanceAnnotation = InitAnnotationBase(distanceAnnotation) as DistanceAnnotation;
             return distanceAnnotation;
         }
 
-        protected new AnnotationInfo InitAnnotationInfo()
+        protected new AnnotationBase InitAnnotationBase(AnnotationBase annotationBase)
         {
-            AnnotationInfo distanceAnnotation = base.InitAnnotationInfo();
+            AnnotationBase distanceAnnotation = base.InitAnnotationBase(annotationBase);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename 'distanceAnnotation' which hides the field with the same name.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)